src/cpu/smp.rs

```diff
@@ -243,14 +243,12 @@ pub fn smp_run_bios_vmpl() -> bool {
             return false;
         }
 
-        let vmsa_va: VirtAddr = match pgtable_map_pages_private(vmsa_pa, PAGE_SIZE) {
+        let vmsa_map: MapGuard = match MapGuard::new_private(vmsa_pa, PAGE_SIZE, true) {
```

Comment:
Because almost all invocations to this function will have `unmap_on_drop=true`, let's assume that's the case and have other function `new_private_persistent()` that sets it to false.